ocsci/tests/test_main.py

```diff
@@ -15,31 +26,25 @@ def test_help(self):
         assert '--cluster-path' in result
         assert '--cluster-name' in result
 
+    @mock.patch('ocsci.main.pytest.main')
+    @mock.patch.object(config, 'update')
+    def test_no_args(self, config_update, pytest_main, testdir):
```

Comment:
@zmc I am not familiar with mock, so I have tried to understand this and hope I understand this test right.

This mocking the ocsci.main.pytest.main, so once you call the main.main([]) without any arguments, you are checking if `update` on mocked `update` function from `config` module was called, but as you changed logic in the main module on [line 33](https://github.com/red-hat-storage/ocs-ci/pull/259/files#diff-3e198ff054d2633195cb6203e4b9cbaaR37) and just  returning in this case you won't do any call on update function and thats the assert on line 33 is about to test if no call of update function has been done. Right?

Maybe some description of this test would be good to understand of this test.

Reply:
That's correct! Here, we're just making sure that `update()` isn't called unnecessarily.